openbas-front/src/utils/String.js

```diff
@@ -94,3 +97,63 @@ export const formatMacAddress = (mac) => {
 export const formatIp = (ip) => {
   return ip.toUpperCase();
 };
+
+export const INJECT = 'inject';
+export const SIMULATION = 'simulation';
+export const SCENARIO = 'scenario';
+export const ATOMIC_BASE_URL = '/admin/atomic_testings';
+export const SIMULATION_BASE_URL = '/admin/simulations';
+export const SCENARIO_BASE_URL = '/admin/scenarios';
+
+const renderLink = (title, url) => (
+  <Tooltip title={title}>
+    <MUILink
+      component={Link}
+      to={url}
+      underline="hover"
```

Comment:
To fit with this screen it woule be better to remove this line
![image](https://github.com/user-attachments/assets/c96f66e5-d8bf-4d90-8eb4-e0cdb42e01f0)